docs/general/builtins/window-functions.rst

```diff
@@ -48,6 +48,13 @@ If ``FILTER`` is specified, then only the rows that met the :ref:`WHERE
 functions that are :ref:`aggregates <aggregation>` accept the ``FILTER``
 clause.
 
+If ``IGNORE NULLS`` flag is specified, then the null values are excluded from
+the window function executions. The window functions that support the flags
+are: :ref:`window-functions-lead`, :ref:`window-functions-lag`,
+:ref:`window-functions-first-value`, :ref:`window-functions-last-value`,
+and :ref:`window-functions-nth-value`. If no flags are specified, then
+``RESPECT NULLS`` is set by default.
```

Comment:
> If no flags are specified, then ``RESPECT NULLS`` is set by default.

This would mean it should be possible to normalize the `Boolean` for `ignoreNull` to a `boolean` in the AstVisitor by translating a `null` to `false`, no?


This sentence to me would also imply that `RESPECT NULLS` should always work, but for some functions it will cause a failure

Reply:
> This would mean it should be possible to normalize the Boolean for ignoreNull to a boolean in the AstVisitor by translating a null to false, no?

the SQL standard states that ignore/respect nulls are acceptable only to the five window functions: first_value, last_value, nth_value, lead, and lag. But there are aggregates or rank, row_number, etc that are still treated as window functions but should not accept the flag. So, need the null to to guard against for those that should not accept it.

Reply:
I could edit the docs to be more clear, `If no flags are specified to the five previously stated window functions, then ``RESPECT NULLS`` is set by default.`

Reply:
Or something like "If a function supports this option and it isn't specified, it defaults to ``RESPECT NULLS``"